nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LookupRecord.java

```diff
@@ -196,6 +198,18 @@ public class LookupRecord extends AbstractProcessor {
         .required(true)
         .build();
 
+    static final PropertyDescriptor RESULT_RECORD_PATH = new PropertyDescriptor.Builder()
+        .name("result-record-path")
+        .displayName("Result RecordPath")
```

Comment:
Isn't the pattern for new properties to have the `.name` and `.displayName` be the same
```suggestion
        .name("Result RecordPath")
        .displayName("Result RecordPath")
```

Reply:
@dan-s1 Yes, but this is just moving the descriptor declaration `RESULT_RECORD_PATH`, so it isn't introducing a new property. See above changes on lines 158-163.